Python/ds/AVL_Tree.py

```diff
@@ -0,0 +1,236 @@
+"""
+AVL tree is a self-balancing Binary Search Tree (BST) where the difference between heights of left and right subtrees cannot be more than one for all nodes and if at any time they differ by more than one, rebalancing is done to restore this property. Each node maintains extra information called a balance factor whose value is either -1, 0 or +1 where balance factor of a node in an AVL tree is the difference between the height of the left subtree and that of the right subtree of that node.
```

Comment:
Justify align this docstring.

Reply:
@atarax665 Can you please tell me how to justify align docstring in the python file? I was unable to find a proper solution for this and thus brought the lines one below the other.